product_supplierinfo_for_customer_sale/tests/test_product_name_search.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+
+from openerp.tests.common import TransactionCase
```

Comment:
`.common` is not longer required just use `from openerp.tests import TransactionCase`